tests/broker-database/dedicated-connection.robot

```diff
@@ -46,4 +46,13 @@ DEDICATED_DB_CONNECTION_${nb_conn}_${store_in_data_bin}
     ...    2    2    yes
     ...    3    3    yes
     ...    3    2    no
-    [Teardown]    Stop Broker    only_central=${True}
+
+
+*** Keywords ***
+Test Clean
+    TRY
+        Kindly Stop Broker    only_central=${True}
+    EXCEPT
+        Log    can't kindly stop broker
+    END
+    Save logs If Failed
```

Comment:
It would be more interesting if it were put in the resources.robot file so that it could be used in all the test sheets.